api/src/main/java/org/pmiops/workbench/actionaudit/auditors/ProfileAuditorImpl.kt

```diff
@@ -64,7 +64,7 @@ constructor(
                             agentIdMaybe = userProvider.get().userId,
                             agentEmailMaybe = userProvider.get().username,
                             targetType = TargetType.PROFILE,
-                            targetIdMaybe = userProvider.get().userId,
```

Comment:
Now shared between updateProfile and adminUpdateProfile - so we can't assume the target is the caller

Reply:
Good catch.

Thinking out loud though, I wonder if it would ever make sense to have a targetUserProvider for administrative methods. I.e. it would provide the administrator doing the call (if one exists), otherwise null (or we can try the OptionalProvider, but CH wasn't too excited about that last time).

In other words, I rely on Spring context for things like this: info that changes globally.